### PR TITLE
README additions (WIP)

### PR DESCRIPTION
**Pull request by lucperkins**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -33,15 +35,28 @@ Coming soon.
...
+The site is published automatically by [Netlify](https://www.netlify.com/) whenever changes are merged to the `master` branch. The site cannot be published in an ad-hoc way (e.g. through a `make` command or script in the repo).
+
+## Contributing to the site
+
+We strongly encourage you to contribute to this site!! There are several ways you can do so:
+
+* By filing [issues](issues) if you see errors, would like to request content, etc.
+* By submitting [pull requests](pulls) with desired changes
+
+If you submit a pull request, there are two things to be aware of:
```
Comment (yurishkuro) [inline README.md]: I think the above addition to Contributing section is good, but don't remove the link to the full CONTRIBUTING doc, which has specific instructions.

Comment (lucperkins) [reply README.md]: BTW, this is just a WIP

**Comment by yurishkuro** [comment]
@lucperkins is this ready to merge?

**Comment by lucperkins** [comment]
For now, yes. I’ll eventually do a much more thorough guide for contributors.

**Pull request merged**